src/treemodel.cpp

```diff
@@ -454,10 +454,12 @@ void TreeModel::SetupChild(TreeItem *child, const QJsonObject & event)
     child->SetData(COL::Site, event["site"].toString());
     child->SetData(COL::User, event["user"].toString());
     child->SetData(COL::Key, event["k"].toString());
+    bool hasArtData = false;
     if (event.contains("a"))
```

Comment:
reads easier for me:
```
bool hasArtData = event.contains("a");
if (hasArtData)
```

Reply:
I completely agree, I will make the change